modules/notes/assets/js/editor/module.js

```diff
@@ -0,0 +1,47 @@
+class Module extends elementorModules.editor.utils.Module {
+	onInit() {
+		this.contextMenuNotesGroup();
+	}
+
+	contextMenuNotesGroup() {
+		const elTypes = [ 'widget', 'section', 'column', 'container' ];
+
+		elTypes.forEach( ( type ) => {
+			elementor.hooks.addFilter( `elements/${ type }/contextMenuGroups`, this.contextMenuAddGroup );
+		} );
```

Comment:
Maybe add a filter for all of the elements, so you'll have a single filter?
```suggestion
		elementor.hooks.addFilter( `elements/contextMenuGroups`, this.contextMenuAddGroup );
```

Check with @avivu or @shilo-ey if it fine from their side

Reply:
I thought about it when working on it, but I am not sure how good of an idea that is, especially that there are so many places that already have custom mods to the contextMenu.
Touching it (or even starting a discussion over it) might make it from a tiny task into a big thing.
I do agree it's a good idea, but maybe as a TODO for later?

Reply:
You don't need to touch the other places, you're just adding a new API.
Anyway, yeah, maybe we should have a new task for that to create that API and update all of the places that uses the old one.

Reply:
Not super complicated, but also will require some development and some testing to do.
I don't want to mix with the current task. Created a new one for it.